Bug: vsh lint 报错了

### Version

Vben Admin V5

### Describe the bug?

```shell
git add .
pnpm commit
```

```shell
> vben-admin-monorepo@5.7.0 lint /Users/elkon/Desktop/wrokspace/vue-vben-admin
> vsh lint

Checking formatting...

All matched files use the correct format.
Finished in 4867ms on 1835 files using 2 threads.

  × eslint(no-console): Unexpected console statement.
    ╭─[scripts/clean.mjs:32:7]
 31 │       await fs.rm(itemPath, { force: true, recursive: true });
 32 │       console.log(`✅ Deleted: ${itemPath}`);
    ·       ───────────
 33 │       return false; // 已删除，无需递归
    ╰────
  help: Supported methods are: warn, error.

  × eslint(no-console): Unexpected console statement.
     ╭─[scripts/clean.mjs:119:3]
 118 │ 
 119 │   console.log(
     ·   ───────────
 120 │     `🚀 Starting cleanup of targets: ${cleanupTargets.join(', ')} from root: ${rootDir}`,
     ╰────
  help: Supported methods are: warn, error.

  × eslint(no-console): Unexpected console statement.
     ╭─[scripts/clean.mjs:127:5]
 126 │     // 先统计要删除的目标数量
 127 │     console.log('📊 Scanning for cleanup targets...');
     ·     ───────────
 128 │ 
     ╰────
  help: Supported methods are: warn, error.

  × eslint(no-console): Unexpected console statement.
     ╭─[scripts/clean.mjs:134:5]
 133 │ 
 134 │     console.log(
     ·     ───────────
 135 │       `✨ Cleanup process completed successfully in ${duration.toFixed(2)}s`,
     ╰────
  help: Supported methods are: warn, error.

  × unicorn(no-process-exit): Don't use `process.exit()`
     ╭─[scripts/clean.mjs:139:5]
 138 │     console.error(`💥 Unexpected error during cleanup: ${error.message}`);
 139 │     process.exit(1);
     ·     ───────────────
 140 │   }
     ╰────
  help: Throw an error instead.

  × unicorn(no-process-exit): Don't use `process.exit()`
    ╭─[scripts/vsh/src/index.ts:55:7]
 54 │       );
 55 │       process.exit(1);
    ·       ───────────────
 56 │     }
    ╰────
  help: Throw an error instead.

  × unicorn(no-process-exit): Don't use `process.exit()`
    ╭─[scripts/vsh/src/index.ts:65:5]
 64 │     );
 65 │     process.exit(1);
    ·     ───────────────
 66 │   }
    ╰────
  help: Throw an error instead.

  × unicorn(no-process-exit): Don't use `process.exit()`
    ╭─[scripts/vsh/src/index.ts:76:3]
 75 │   );
 76 │   process.exit(1);
    ·   ───────────────
 77 │ });
    ╰────
  help: Throw an error instead.

  × unicorn(no-process-exit): Don't use `process.exit()`
    ╭─[scripts/turbo-run/src/run.ts:13:5]
 12 │     console.error('Please enter the command to run');
 13 │     process.exit(1);
    ·     ───────────────
 14 │   }
    ╰────
  help: Throw an error instead.

  × unicorn(no-process-exit): Don't use `process.exit()`
    ╭─[scripts/turbo-run/src/run.ts:38:7]
 37 │       cancel('👋 Has cancelled');
 38 │       process.exit(0);
    ·       ───────────────
 39 │     }
    ╰────
  help: Throw an error instead.

  × unicorn(no-process-exit): Don't use `process.exit()`
    ╭─[scripts/turbo-run/src/run.ts:46:5]
 45 │     console.error('No app found');
 46 │     process.exit(1);
    ·     ───────────────
 47 │   }
    ╰────
  help: Throw an error instead.

  × unicorn(no-process-exit): Don't use `process.exit()`
    ╭─[scripts/turbo-run/src/index.ts:22:3]
 21 │   consola.error(error);
 22 │   process.exit(1);
    ·   ───────────────
 23 │ }
    ╰────
  help: Throw an error instead.

  × vue(prefer-import-from-vue): Use `import from 'vue'` instead of `import from '@vue/*'`.
   ╭─[packages/@core/base/shared/src/utils/inference.ts:1:48]
 1 │ import { isFunction, isObject, isString } from '@vue/shared';
   ·                                                ─────────────
 2 │ 
   ╰────
  help: Replace `'@vue/shared'` with `'vue'`.

  × vue(prefer-import-from-vue): Use `import from 'vue'` instead of `import from '@vue/*'`.
   ╭─[packages/@core/base/shared/src/utils/inference.ts:1:48]
 1 │ import { isFunction, isObject, isString } from '@vue/shared';
   ·                                                ─────────────
 2 │ 
   ╰────
  help: Replace `'@vue/shared'` with `'vue'`.

  × vue(prefer-import-from-vue): Use `import from 'vue'` instead of `import from '@vue/*'`.
   ╭─[packages/@core/base/shared/src/utils/inference.ts:1:48]
 1 │ import { isFunction, isObject, isString } from '@vue/shared';
   ·                                                ─────────────
 2 │ 
   ╰────
  help: Replace `'@vue/shared'` with `'vue'`.

  × vue(prefer-import-from-vue): Use `import from 'vue'` instead of `import from '@vue/*'`.
   ╭─[packages/@core/base/shared/src/utils/inference.ts:1:48]
 1 │ import { isFunction, isObject, isString } from '@vue/shared';
   ·                                                ─────────────
 2 │ 
   ╰────
  help: Replace `'@vue/shared'` with `'vue'`.

  × vue(prefer-import-from-vue): Use `import from 'vue'` instead of `import from '@vue/*'`.
   ╭─[packages/@core/base/shared/src/utils/inference.ts:1:48]
 1 │ import { isFunction, isObject, isString } from '@vue/shared';
   ·                                                ─────────────
 2 │ 
   ╰────
  help: Replace `'@vue/shared'` with `'vue'`.

  × vue(prefer-import-from-vue): Use `import from 'vue'` instead of `import from '@vue/*'`.
   ╭─[packages/@core/base/shared/src/utils/inference.ts:1:48]
 1 │ import { isFunction, isObject, isString } from '@vue/shared';
   ·                                                ─────────────
 2 │ 
   ╰────
  help: Replace `'@vue/shared'` with `'vue'`.

  × unicorn(no-process-exit): Don't use `process.exit()`
     ╭─[scripts/vsh/src/publint/index.ts:171:15]
 170 │     );
 171 │     !check && process.exit(1);
     ·               ───────────────
 172 │   } else {
     ╰────
  help: Throw an error instead.

  × unicorn(no-process-exit): Don't use `process.exit()`
    ╭─[internal/node-utils/scripts/build.mjs:38:5]
 37 │   if (result.status !== 0) {
 38 │     process.exit(result.status ?? 1);
    ·     ────────────────────────────────
 39 │   }
    ╰────
  help: Throw an error instead.

  × import(no-self-import): A module importing itself is not allowed
   ╭─[packages/@core/ui-kit/menu-ui/src/sub-menu.vue:7:21]
 6 │ import { MenuBadge, MenuItem, SubMenu as SubMenuComp } from './components';
 7 │ import SubMenu from './sub-menu.vue';
   ·                     ────────────────
 8 │ 
   ╰────
  help: Remove this import. A module should not import itself.

Found 0 warnings and 21 errors.
Finished in 5.5s on 1143 files with 226 rules using 2 threads.

 ERROR  An unexpected error occurred: 
 Command failed with exit code 1: oxlint '--threads=2'

 ELIFECYCLE  Command failed with exit code 1.

exit status 1
  ────────────────────────────────────
summary: (done in 5.97 seconds)
🥊 checkType (5.41 seconds)
🥊 lint (5.94 seconds)
git exited with error code 1
 ELIFECYCLE  Command failed with exit code 1.
```

### Reproduction

null

### System Info

```shell
all
```

### Relevant log output

```shell

```

### Validations

- [x] Read the [docs](https://doc.vben.pro/)
- [x] Ensure the code is up to date. (Some issues have been fixed in the latest version)
- [x] I have searched the [existing issues](https://github.com/vbenjs/vue-vben-admin/issues) and checked that my issue does not duplicate any existing issues.
- [x] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vbenjs/vue-vben-admin/discussions) or join our [Discord Chat Server](https://discord.gg/8GuAdwDhj6).
- [x] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

同样全新克隆，仓库原代码lint无法通过
 
1. 下载最新主分支
2. pnpm install安装依赖
3. 添加空白文件aaa
4. git添加aaa并commit
5. commit预检钩子运行通过lefthook运行lint检查，无法通过

```bash
# admin @ matrix in ~/Coding/Leading [16:46:03] 
$ git clone https://github.com/vbenjs/vue-vben-admin.git 
Cloning into 'vue-vben-admin'...
remote: Enumerating objects: 79207, done.
remote: Counting objects: 100% (200/200), done.
remote: Compressing objects: 100% (169/169), done.
remote: Total 79207 (delta 83), reused 36 (delta 31), pack-reused 79007 (from 3
Receiving objects: 100% (79207/79207), 45.46 MiB | 4.06 MiB/s, done.
Resolving deltas: 100% (48999/48999), done.

# admin @ matrix in ~/Coding/Leading [16:46:32] 
$ cd vue-vben-admin 

# admin @ matrix in ~/Coding/Leading/vue-vben-admin on git:main o [16:46:33] 
$ ls
apps               oxlint.config.ts     README.zh-CN.md
cspell.json        package.json         scripts
docs               packages             stylelint.config.mjs
eslint.config.mjs  playground           tea.yaml
internal           pnpm-lock.yaml       turbo.json
lefthook.yml       pnpm-workspace.yaml  vben-admin.code-workspace
LICENSE            README.ja-JP.md      vitest.config.ts
oxfmt.config.ts    README.md

# admin @ matrix in ~/Coding/Leading/vue-vben-admin on git:main o [16:46:33] 
$ pnpm install
Scope: all 46 workspace projects
Lockfile is up to date, resolution step is skipped
Packages: +1734
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Progress: resolved 1734, reused 1733, downloaded 0, added 1734, done
node_modules/.pnpm/@parcel+watcher@2.5.6/node_modules/@parcel/watcher: Running install script...
node_modules/.pnpm/core-js@3.49.0/node_modules/core-js: Running postinstall script...
node_modules/.pnpm/@parcel+watcher@2.5.6/node_modules/@parcel/watcher: Running inode_modules/.pnpm/core-js@3.49.0/node_modules/core-js: Running postinstall scrinode_modules/.pnpm/vue-demi@0.14.10_vue@3.5.35_typescript@6.0.3_/node_modules/vunode_modules/.pnpm/lefthook@2.1.9/node_modules/lefthook: Running postinstall script, done in 360ms

devDependencies:
+ @changesets/changelog-github 0.7.0
+ @changesets/cli 2.31.0
+ @tsdown/css 0.22.1
+ @types/node 25.9.1
+ @vben/commitlint-config 5.7.0 <- internal/lint-configs/commitlint-config
+ @vben/eslint-config 5.7.0 <- internal/lint-configs/eslint-config
+ @vben/oxfmt-config 5.7.0 <- internal/lint-configs/oxfmt-config
+ @vben/oxlint-config 5.7.0 <- internal/lint-configs/oxlint-config
+ @vben/stylelint-config 5.7.0 <- internal/lint-configs/stylelint-config
+ @vben/tailwind-config 5.7.0 <- internal/tailwind-config
+ @vben/tsconfig 5.7.0 <- internal/tsconfig
+ @vben/turbo-run 5.7.0 <- scripts/turbo-run
+ @vben/vite-config 5.7.0 <- internal/vite-config
+ @vben/vsh 5.7.0 <- scripts/vsh
+ @vitejs/plugin-vue 6.0.7
+ @vitejs/plugin-vue-jsx 5.1.5
+ cross-env 10.1.0
+ cspell 10.0.1
+ eslint 10.4.1
+ happy-dom 20.9.0
+ is-ci 4.1.0
+ lefthook 2.1.9
+ oxfmt 0.53.0
+ oxlint 1.68.0
+ oxlint-tsgolint 0.23.0
+ playwright 1.60.0
+ rimraf 6.1.3
+ tailwindcss 4.3.0
+ tsdown 0.22.1
+ turbo 2.9.16
+ typescript 6.0.3
+ vite 8.0.10
+ vitest 4.1.8
+ vue 3.5.35
+ vue-tsc 3.3.3

. postinstall$ pnpm -r run --if-present stub
[89 lines collapsed]
│ internal/lint-configs/eslint-config stub: ℹ dist/index.mjs   25.84 kB │ gzi…
│ internal/lint-configs/eslint-config stub: ℹ dist/index.d.ts   0.21 kB │ gzi…
│ internal/lint-configs/eslint-config stub: ℹ 2 files, total: 26.05 kB
│ internal/lint-configs/eslint-config stub: ✔ Build complete in 930ms
│ internal/lint-configs/eslint-config stub: Done
│ internal/vite-config stub: ℹ dist/index.mjs   30.16 kB │ gzip: 9.64 kB
│ internal/vite-config stub: ℹ dist/index.d.ts   4.92 kB │ gzip: 1.32 kB
│ internal/vite-config stub: ℹ 2 files, total: 35.08 kB
│ internal/vite-config stub: ✔ Build complete in 1550ms
│ internal/vite-config stub: Done
└─ Done in 4.3s
. prepare$ is-ci || pnpm exec lefthook install
│ sync hooks: ✔️(pre-commit, post-merge, commit-msg)
└─ Done in 646ms
Done in 21s using pnpm v11.5.0

# admin @ matrix in ~/Coding/Leading/vue-vben-admin on git:main o [16:46:57] 
$ ls
apps               oxfmt.config.ts      README.md
cspell.json        oxlint.config.ts     README.zh-CN.md
docs               package.json         scripts
eslint.config.mjs  packages             stylelint.config.mjs
internal           playground           tea.yaml
lefthook.yml       pnpm-lock.yaml       turbo.json
LICENSE            pnpm-workspace.yaml  vben-admin.code-workspace
node_modules       README.ja-JP.md      vitest.config.ts

# admin @ matrix in ~/Coding/Leading/vue-vben-admin on git:main o [16:47:02] 
$ touch aaa

# admin @ matrix in ~/Coding/Leading/vue-vben-admin on git:main x [16:47:07] 
$ git st
g## main...origin/main
?? aaa

# admin @ matrix in ~/Coding/Leading/vue-vben-admin on git:main x [16:47:08] 
$ git add .

# admin @ matrix in ~/Coding/Leading/vue-vben-admin on git:main x [16:47:09] 
$ git st
## main...origin/main
A  aaa

# admin @ matrix in ~/Coding/Leading/vue-vben-admin on git:main x [16:47:10] 
$ git cm 'feat: test'
╭─────────────────────────────────────────╮
│ 🥊 lefthook  v2.1.9   hook:  pre-commit │
╰─────────────────────────────────────────╯
┃  checkType ❯ 
$ turbo run typecheck
• turbo 2.9.16

   • Packages in scope: @vben-core/composables, @vben-core/design, @vben-core/form-ui, @vben-core/icons, @vben-core/layout-ui, @vben-core/menu-ui, @vben-core/popup-ui, @vben-core/preferences, @vben-core/shadcn-ui, @vben-core/shared, @vben-core/tabs-ui, @vben-core/typings, @vben/access, @vben/backend-mock, @vben/commitlint-config, @vben/common-ui, @vben/constants, @vben/docs, @vben/eslint-config, @vben/hooks, @vben/icons, @vben/layouts, @vben/locales, @vben/node-utils, @vben/oxfmt-config, @vben/oxlint-config, @vben/playground, @vben/plugins, @vben/preferences, @vben/request, @vben/stores, @vben/stylelint-config, @vben/styles, @vben/tailwind-config, @vben/tsconfig, @vben/turbo-run, @vben/types, @vben/utils, @vben/vite-config, @vben/vsh, @vben/web-antd, @vben/web-antdv-next, @vben/web-ele, @vben/web-naive, @vben/web-tdesign
   • Running typecheck in 45 packages
   • Remote caching disabled

@vben/web-ele:typecheck: cache miss, executing 0f197628621eca58
@vben/web-antd:typecheck: cache miss, executing ce949d3fcf5432ba
@vben/web-tdesign:typecheck: cache miss, executing 13b6798a35365aad
@vben/web-antdv-next:typecheck: cache miss, executing 4d84044556443394
@vben/web-naive:typecheck: cache miss, executing 4fda07a7035f06b3
@vben/playground:typecheck: cache miss, executing 77e70fdb51df5a84
@vben/web-antdv-next:typecheck: $ vue-tsc --noEmit --skipLibCheck
@vben/web-ele:typecheck: $ vue-tsc --noEmit --skipLibCheck
@vben/web-tdesign:typecheck: $ vue-tsc --noEmit --skipLibCheck
@vben/web-naive:typecheck: $ vue-tsc --noEmit --skipLibCheck
@vben/playground:typecheck: $ vue-tsc --noEmit --skipLibCheck
@vben/web-antd:typecheck: $ vue-tsc --noEmit --skipLibCheck

 Tasks:    6 successful, 6 total
Cached:    0 cached, 6 total
  Time:    38.391s 


┃  lint ❯ 
$ vsh lint
Checking formatting...

All matched files use the correct format.
Finished in 9123ms on 1729 files using 2 threads.
Found 0 warnings and 0 errors.
Finished in 13.2s on 1031 files with 238 rules using 2 threads.

/Users/admin/Coding/Leading/vue-vben-admin/packages/@core/ui-kit/shadcn-ui/src/ui/dialog/DialogOverlay.vue
  4:1  error  Use `export…from` to re-export `default`  unicorn/prefer-export-from

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.


 ERROR  An unexpected error occurred: 
 Command failed with exit code 1: eslint . --cache

[ELIFECYCLE] Command failed with exit code 1.

exit status 1
  ────────────────────────────────────
summary: (done in 45.86 seconds)
✔️ checkType (39.76 seconds)
🥊 lint (45.82 seconds)

# admin @ matrix in ~/Coding/Leading/vue-vben-admin on git:main x [16:47:59] C:1
$ node -v
v22.22.0
```

>   System:
>     OS: macOS 15.7.7
>     CPU: (8) arm64 Apple M1 Pro
>     Memory: 691.75 MB / 16.00 GB
>     Shell: 5.9 - /bin/zsh
>   Binaries:
>     Node: 22.22.0 - /Users/admin/.nodenv/versions/22.22.0/bin/node
>     npm: 10.9.4 - /Users/admin/.nodenv/versions/22.22.0/bin/npm
>     pnpm: 11.5.0 - /opt/homebrew/bin/pnpm
>   Browsers:
>     Chrome: 149.0.7827.54
>     Firefox: 151.0.3
>     Safari: 26.5

--- a/apps/web-antdv-next/src/preferences.ts
+++ b/apps/web-antdv-next/src/preferences.ts
@@ -1,4 +1,7 @@
-import { appCopyrightPreferences, defineOverridesPreferences } from '@vben/preferences';
+import {
+  appCopyrightPreferences,
+  defineOverridesPreferences,
+} from '@vben/preferences';
 
 /**
  * @description 项目配置文件

--- a/apps/web-ele/src/preferences.ts
+++ b/apps/web-ele/src/preferences.ts
@@ -1,4 +1,7 @@
-import { appCopyrightPreferences, defineOverridesPreferences } from '@vben/preferences';
+import {
+  appCopyrightPreferences,
+  defineOverridesPreferences,
+} from '@vben/preferences';
 
 /**
  * @description 项目配置文件

--- a/apps/web-naive/src/preferences.ts
+++ b/apps/web-naive/src/preferences.ts
@@ -1,4 +1,7 @@
-import { appCopyrightPreferences, defineOverridesPreferences } from '@vben/preferences';
+import {
+  appCopyrightPreferences,
+  defineOverridesPreferences,
+} from '@vben/preferences';
 
 /**
  * @description 项目配置文件

--- a/apps/web-tdesign/src/preferences.ts
+++ b/apps/web-tdesign/src/preferences.ts
@@ -1,4 +1,7 @@
-import { appCopyrightPreferences, defineOverridesPreferences } from '@vben/preferences';
+import {
+  appCopyrightPreferences,
+  defineOverridesPreferences,
+} from '@vben/preferences';
 
 /**
  * @description 项目配置文件

--- a/packages/preferences/src/index.ts
+++ b/packages/preferences/src/index.ts
@@ -28,6 +28,10 @@ const appCopyrightPreferences = {
   icpLink: 'https://beian.miit.gov.cn/',
 } satisfies DeepPartial<Preferences>['copyright'];
 
-export { appCopyrightPreferences, defineOverridesPreferences, definePreferencesExtension };
+export {
+  appCopyrightPreferences,
+  defineOverridesPreferences,
+  definePreferencesExtension,
+};
 
 export * from '@vben-core/preferences';

